code/testapp/src/main/java/com/adobe/marketing/mobile/messagingsample/MessagingApplication.kt

```diff
@@ -22,10 +22,10 @@ import com.adobe.marketing.mobile.edge.identity.Identity
 import com.google.firebase.messaging.FirebaseMessaging
 
 class MessagingApplication : Application() {
-    private val ENVIRONMENT_FILE_ID = "3149c49c3910/4f6b2fbf2986/launch-7d78a5fd1de3-development"
-    private val ASSURANCE_SESSION_ID = ""
+    private val ENVIRONMENT_FILE_ID = "3149c49c3910/473386a6e5b0/launch-6099493a8c97-development"
```

Comment:
nit: can we store the existing app id in another val? it will save time in the future if we need to go back to different testing.

Reply:
noticed that the staging flag is set to true. is this appid a staging id?

Reply:
This is prod app id for a different tag property. I accidentally changed it, will revert it back